.vscode/launch.json

```diff
@@ -0,0 +1,37 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug tests",
+      "program": "${workspaceRoot}/node_modules/protractor/bin/protractor",
+      "args": ["${workspaceRoot}/dist/protractor/local.config.js"],
+      "preLaunchTask": "npm: build",
+      "sourceMaps": true,
+      "smartStep": true,
+      "internalConsoleOptions": "openOnSessionStart",
+      "outFiles": [
+          "${workspaceFolder}/dist/**/*.js"
+      ]
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug headless tests",
+      "program": "${workspaceRoot}/node_modules/protractor/bin/protractor",
+      "args": ["${workspaceRoot}/dist/protractor/headless.config.js"],
+      "preLaunchTask": "npm: build",
+      "sourceMaps": true,
+      "smartStep": true,
+      "internalConsoleOptions": "openOnSessionStart",
+      "outFiles": [
+          "${workspaceFolder}/dist/**/*.js"
+      ]
+    }
+  ]
+}
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon:
![image](https://user-images.githubusercontent.com/10209180/126420285-15f4d88e-9a11-40c2-8eda-d3d4634cb332.png)